{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "0dab8dd9_347ac04d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000429
      },
      "writtenOn": "2023-01-20T01:42:12Z",
      "side": 1,
      "message": "i still get error \"unknown type\" with this.",
      "revId": "c50b89f01ee4048bed15f5b85dee3b66fcba29c4",
      "serverId": "3162e3d8-d894-470e-abd9-724f7b0effad"
    }
  ]
}